test/integration/http_server_options_test.lua

```diff
@@ -165,3 +166,25 @@ g.test_idle_timeout_is_unset = function(g)
     t.assert_equals(conn_is_opened, true)
     t.assert_equals(conn_is_closed, false) -- Connection is alive.
 end
+
+g.before_test('test_log_requests_default', function()
+    g.httpd = helpers.cfgserv()
+    g.httpd:start()
+end)
+
+g.test_log_requests_default = function(g)
+    local httpd = g.httpd
+    t.assert_equals(httpd.options.log_requests, false, "Default log_requests should be false")
+end
+
+g.before_test('test_log_requests_custom', function()
+    g.httpd = helpers.cfgserv({
+        log_requests = log.verbose
+    })
+    g.httpd:start()
+end)
+
+g.test_log_requests_custom = function(g)
+    local httpd = g.httpd
+    t.assert_equals(httpd.options.log_requests, log.verbose)
+end
```

Comment:
We already have tests for `log_request` option of `http` module [here](https://github.com/tarantool/http/blob/master/test/integration/http_log_requests_test.lua).